modules/microshift-ingress-controller-config.adoc

```diff
@@ -0,0 +1,153 @@
+
+// Module included in the following assemblies:
+//
+// * microshift_configuring/microshift-ingress-controller.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-ingress-control-config_{context}"]
+= Configuring ingress control in {microshift-short}
+
+You can use detailed ingress control settings by updating the {microshift-short} service configuration file.
+
+.Prerequisites
+
+* You installed the OpenShift CLI (`oc`).
+* You have root access to the cluster.
+* Your cluster uses the OVN-Kubernetes network plugin.
+//Q: Any other prereqs?
+
+.Procedure
+
+. You can apply ingress control settings in either of the two following places:
+
+.. Update the {microshift-short} `config.yaml` configuration file by making a copy of the provided `config.yaml.default` file in the `/etc/microshift/` directory.
+
+.. Use a configuration snippet to apply the ingress control settings you want. To do this, create a configuration snippet YAML file and put it in the the `/etc/microshift/config.d/` configuration directory.
+
```

Comment:
There are some indentation issues with this note, causing the next step to restart the order as step 1. At the very least, I think there needs to be a + added right above the note: 
```suggestion
+
```

However, I think that would indent the note under step 1b, but it seems like the note applies to both substeps. I don't know how you'd want to handle that, but at least this way the indentation and number ordering is valid